src/main/webapp/WEB-INF/tags/instructor/feedbacks/feedbackSessionsFormAdditionalSettingsModal.tag

```diff
@@ -0,0 +1,20 @@
+<%@ tag trimDirectiveWhitespaces="true" %>
+<%@ tag description="instructorFeedbacks - new feedback session form additional settings information modal" pageEncoding="UTF-8" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+
+<%@ attribute name="settingId" required="true"%>
+<%@ attribute name="settingDescription" required="true"%>
+<%@ attribute name="settingOptions" type="java.lang.Object" required="true"%>
```

Comment:
was using primitive array initially whose direct superclass is top level Object class. Changed to collections now.

Reply:
Any reason why you must use the superclass instead of using the class directly?

Reply:
My bad. I was initially trying `[Ljava.lang.String` and IntelliJ wasn't able to resolve it but just realized i could use `java.lang.String[]`. Anyway I changed to Collection since we're using it everywhere else in tag files.